packages/extension/src/newtab/App.tsx

```diff
@@ -94,6 +97,12 @@ function InternalApp({
     }
   }, [contentScriptGranted]);
 
+  useEffect(() => {
+    document.title = unreadCount
```

Comment:
Why do we need to set it manually here?

Reply:
The other implementations are only for the webapp, the extension is not affected by those changes using `<NextSeo />` - this one is specific for the extension as they work a bit differently.